src/Shared/Constants.cs

```diff
@@ -58,12 +58,12 @@ internal static class MSBuildConstants
         /// <summary>
         /// The most current Visual Studio Version known to this version of MSBuild.
         /// </summary>
-        internal const string CurrentVisualStudioVersion = "15.0";
+        internal const string CurrentVisualStudioVersion = "16.0";
 
         /// <summary>
         /// The most current ToolsVersion known to this version of MSBuild.
         /// </summary>
-        internal const string CurrentToolsVersion = CurrentVisualStudioVersion;
+        internal const string CurrentToolsVersion = "15.0";
```

Comment:
This was the line I had missed that caused it to fail a bajillion tests. The fix was . . . not hard.

Reply:
Why is it 15 and not 16?

Reply:
Oh, toolsversion, nvm :)